addons/api/mirage/factories/target.js

```diff
@@ -22,25 +22,23 @@ export default factory.extend({
       'add-credential-sources',
       'remove-credential-sources',
     ],
-
-  id: () => generateId('t_'),
+  id() {
+    return this.type === TYPE_TARGET_TCP
+      ? generateId('ttcp_')
+      : generateId('tssh_');
```

Comment:
Unless we're using the ID prefixes in some way, it's probably overkill to type the mock IDs.

Reply:
👍🏼 I can revert this change.

Reply:
Updated.